src/suricata.c

```diff
@@ -2905,15 +2905,15 @@ int SuricataMain(int argc, char **argv)
         exit(EXIT_FAILURE);
     }
 
-    LandlockSandboxing(&suricata);
-
     SCDropMainThreadCaps(suricata.userid, suricata.groupid);
 
     /* Re-enable coredumps after privileges are dropped. */
     CoredumpEnable();
 
     PreRunPostPrivsDropInit(suricata.run_mode);
 
+    LandlockSandboxing(&suricata);
```

Comment:
this will now run after dropping privs. Does that in any way affect things?

Reply:
No, this is fine, the less privileged user can shut it down from access with landlock without problem.